Exiled.Events/EventArgs/Player/SuccessfullyUsedTapeItemEventArgs.cs

```diff
@@ -0,0 +1,42 @@
+﻿// -----------------------------------------------------------------------
+// <copyright file="SuccessfullyUsedTapeItemEventArgs.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.Events.EventArgs.Player
+{
+    using Exiled.API.Features;
+    using Exiled.API.Features.Items;
+    using Exiled.Events.EventArgs.Interfaces;
+    using InventorySystem.Items;
+
+    /// <summary>
+    /// Contains all information after player uses Tape Item.
+    /// </summary>
+    public class SuccessfullyUsedTapeItemEventArgs : IItemEvent, IPlayerEvent
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="SuccessfullyUsedTapeItemEventArgs"/> class.
+        /// </summary>
+        /// <param name="player"><inheritdoc cref="Player"/></param>
+        /// <param name="itemBase"><inheritdoc cref="Item"/></param>
+        public SuccessfullyUsedTapeItemEventArgs(Player player, ItemBase itemBase)
```

Comment:
I'd personally rename it to
```cs
public UsedTapeEventArgs(Player player, ItemBase itemBase)
```

Reply:
Ah yes, forgot a little how this item work